modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -69,4 +152,45 @@ definition {
 			value1 = "\ENTER");
 	}
 
+	macro resetDefaultValues {
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = ${portlet});
+
+		Click(
+			key_itemName = "Accessibility",
+			locator1 = "ListGroupItem#ITEM_TEXT");
+
+		Click(locator1 = "ObjectAdmin#KEBAB_MENU");
+
+		Click(
+			key_option = "Reset Default Values",
+			locator1 = "JobScheduler#KEBAB_MENU_OPTION");
```

Comment:
Try Icon.ANY_VERTICAL_ELLIPSIS. Use general paths as much as possible from `portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements`  or `uisections` as they try to target clay components that are widely used throughout portal and product teams. Avoid product team specific macros/paths that are not related to the product you are testing.

Reply:
I'm trying to use `MenuItem#ANY_MENU_ITEM`, but it's not working for these items 
![image](https://github.com/liferay-peds/liferay-portal/assets/63858122/139c00d6-6dde-404c-81d9-96348f506ee1)